Cargo.lock

```diff
@@ -190,7 +190,7 @@ dependencies = [
 
 [[package]]
 name = "altair-runtime"
-version = "0.10.31"
+version = "0.10.32"
```

Comment:
@mustermeiszer I guess my editor automatically fixed this issue (https://github.com/centrifuge/centrifuge-chain/pull/1539). For me, you can merge your PR and then we merge this one; or we can merge this one into yours. Whatever you prefer :)

Reply:
Lets merge this one and close mine as this is hopefully all green and needs no force-merge and leaves not another broken commit on main.

Wdyt?

Reply:
Agreed! Thanks 🙏